api/v1beta1/installation_types.go

```diff
@@ -32,6 +32,7 @@ const (
 	InstallationStateObsolete               string = "Obsolete"
 	InstallationStateFailed                 string = "Failed"
 	InstallationStateUnknown                string = "Unknown"
+	InstallationStatePendingChartCreation   string = "PendingChartCreation"
```

Comment:
You probably need to add this new state to Kots so we don't see an unknown there